test/sycl_tests.cc

```diff
@@ -148,4 +148,21 @@ TEST_CASE_METHOD(test_utils::device_queue_fixture, "SYCL can access empty buffer
 	}
 }
 
+#if CELERITY_FEATURE_SIMPLE_SCALAR_REDUCTIONS
+
+TEST_CASE_METHOD(test_utils::device_queue_fixture, "SYCL has working simple scalar reductions", "[sycl][reductions]") {
+	constexpr size_t N = 1024;
```

Comment:
I remember this test being sensitive to `N`. If that's correct we could run it for multiple values of `N` with  `GENERATE`+`CAPTURE`.